core/tests/performance_framework/perf_services.py

```diff
@@ -68,6 +68,15 @@ def __init__(self, browser=DEFAULT_BROWSER_SOURCE):
             error_msg = 'Unsupported browser specified: %s' % browser
             raise ValueError(error_msg)
 
+    def load_url(self, page_url):
+        """Loads the specified url."""
```

Comment:
Intention might be good to specify here, too, to help indicate why a developer might want to use it. E.g.

```python
"""Loads the specified url, caching related resources."""
```